src/blocklypropclient.js

```diff
@@ -408,20 +373,14 @@ function establish_socket() {
             if (ws_msg.type === 'hello-client') {
                 // type: 'hello-client',
                 // version: [String version (semantic versioning)]
-                client_version = version_as_number(ws_msg.version);
-
-                /* Test code: Terminal dump
-                 setInterval(function() {
-                 Terminal dumper!
-                 console.log('Terminal Dump!\n-------------------\n' + terminal_dump);
-                 terminal_dump = null;
-                 }, 10000);
-                 */
+                checkClientVersionModal(ws_msg.version);
 
                 if (window.getURLParameter('debug')) {
                     console.log("Websocket client/launcher found - version " + ws_msg.version);
                 }
 
+                // TODO: Add version checking here.
```

Comment:
Will address in next PR.

Reply:
So we don't include this code in this PR? Let's remove it if we're not going to address it in this PR.

Reply:
You said this PR was full enough, so it's staged in by 200b.